### ISSUE TITLE
Calling Spring @Controller doesn't seem work

### ISSUE THREAD

**Issue by ronnylat** [code8/tapestry-boot#5]
Please advice if Spring @Controller @RestController still available on this framework?

@RestController
public class ImageController {

    @RequestMapping(value="/rest/{id}", method= RequestMethod.GET)
    public String index(@PathVariable( "id" ) String id) {
        return "Greetings from Spring Hello blabla!" + id;
    }
}

**Comment by daveyx** [comment]
yes, this will work, but you have to exclude this path in the appmodule
